howto/application/update-application.md

```diff
@@ -82,6 +85,26 @@ resources:
   disk-size: 8GB
 ```
 
+````
+
+```{group-tab} Dashboard
+
+You can edit an application from the *Applications* list page.
+
+In the guided form, you can customize most of the fields that are available during {ref}`application creation <howto-create-application>`. Some of the fields, like the application name, or the container/VM option, cannot be modified and are disabled. When a field is disabled, hovering over it will display a tooltip explaining the reason.
+
+Similar to the application creation process, you can unlock advanced customization capabilities by toggling the *Customize manifest.yaml* switch at the bottom of the page. Note that customizing the manifest or updating one of the following fields will create a new version of the application:
+* APK
+* {ref}`manifest version name <ref-application-manifest>`
+* {ref}`boot activity <ref-application-manifest>`
+* {ref}`features <ref-feature-flags>` 
+* {ref}`sec-application-manifest-watchdog`
+
```

Comment:
hmm, this is not the case when I check the backend code. 
https://github.com/canonical/ams/blob/main/internal/ams/backend/application.go#L639-L659

Reply:
Sure, when I read the following [code](https://github.com/canonical/ams/blob/main/internal/ams/backend/application.go#L639-L659)
if one of the following fields is given during the application update, there would be a new app version to be created.
- Services
- Watchdog
- BootActivity
- RequiredPermissions
- VideoEncoder
- ManifestVersion
- Features
- Hooks
- Bootstrap

So we may need to extend the list a bit in the doc as currently we only mentioned 4 of them + APK

Reply:
This is about the dashboard, not the CLI nor the AMS API. The only fields that you can customise in the dashboard form are those listed there. The other fields are not included in the form.

Reply:
Okay, thanks for the correction.